tests/modules/text_generation/test_text_generation_tgis.py

```diff
@@ -79,6 +88,35 @@ def test_save_model_can_run():
         StubTGISClient.validate_unary_generate_response(result)
 
 
+def test_local_train_load_tgis():
+    """Check if the model trained in local module is able to
+    be loaded in TGIS module / backend
+    """
+    train_kwargs = {
+        "base_model": HFAutoSeq2SeqLM.bootstrap(
+            model_name=SEQ2SEQ_LM_MODEL, tokenizer_name=SEQ2SEQ_LM_MODEL
+        ),
+        "num_epochs": 1,
+        "train_stream": caikit.core.data_model.DataStream.from_iterable(
+            [
+                GenerationTrainRecord(
+                    input="@foo what a cute dog!", output="no complaint"
+                )
+            ]
+        ),
+        "torch_dtype": torch.float32,
+    }
+    model = TextGeneration.train(**train_kwargs)
```

Comment:
Yeah, it isn't necessary, but I do worry a bit that this will feel kind of strange coming from other caikit implementations, where it's very weird for train on one type to produce a loadable module of a different type. Let's punt that discussion for now though!